csharp/lib/Cargo.toml

```diff
@@ -18,5 +18,6 @@ tokio = { version = "^1", features = ["rt", "macros", "rt-multi-thread", "time"]
 logger_core = {path = "../../logger_core"}
 
 [profile.release]
-lto = true
-debug = true
+lto = "fat"
+debug = false
```

Comment:
debug = false is the default value of the release profile, so it can be removed (in the other cargo.toml files too)